packages/perseus/src/widgets/number-line.tsx

```diff
@@ -472,25 +472,25 @@ class NumberLine extends React.Component<Props, State> {
         // it can't be interacted with.
         let fill;
         if (isOpen) {
-            fill = KhanColors._BACKGROUND;
+            fill = "transparent";
         } else if (props.static) {
-            fill = KhanColors.DYNAMIC;
+            fill = KhanColors.BLUE;
```

Comment:
How would you feel about replacing this (and other uses below) with WB `color.blue` and not importing `KhanColors` at all where possible? I think that would cut out the intermediate step and make it clear exactly where the color is coming from.

Reply:
I think I may have used some of my understanding of backend coding here to make this decision, but I think it might not be the same for frontend. Let me see what you think. I was thinking having one place where we import WB colors would be more efficient than importing it in all the places the colors are used, specifically because WB is an external library. So we're only doing the import here and not all over. Maybe the import still ends up happening in all the places anyway as each time it goes to to the KhanColors file and it ends up importing from WB at that point anyway. 🤔